src/app/core-components/welcome/welcome.component.html

```diff
@@ -22,9 +22,7 @@
 
                             <fa-icon [icon]="faChevronRight"></fa-icon>
                             <p class="card-text ms-2">
-                                Use the setup wizard to create a new assessment. Here you can create a company and facility
-                                to
-                                add your new assessment to or pick from existing ones in your system.
+                                Use the setup wizard to enter data relating to companies, facilities and assessments.
```

Comment:
A little change and add a comma after facilities:
Use the setup wizard to enter data **about the** companies, facilities, and assessments.